qcoverage/src/org/qcmg/coverage/Coverage.java

```diff
@@ -190,25 +188,32 @@ private void saveCoverageReport() throws Exception {
 		for (final CoverageReport report : jobQueue.getCoverageReport()) {
 			stats.getCoverageReport().add(report);
 		}
+		
 		if (options.hasVcfFlag() && options.hasPerFeatureOption()) {
 			writeVCFReport(stats);
 		}
+		
 		if (options.hasXmlFlag()) {
 			writeXMLCoverageReport(stats);
-//			if (options.hasPerFeatureOption())
-//				writeVCFReport(stats);
-		} else if (options.hasPerFeatureOption()) {
+		}
+		
+		if (options.hasTxtFlag() && options.hasPerFeatureOption()) {			 
```

Comment:
Could these 2 if stmt's be combined into 1 stmt? Would aid readability...

Reply:
yes, I will merge them